config/sync/system.site.yml

```diff
@@ -8,7 +8,7 @@ slogan: ''
 page:
   403: ''
   404: ''
-  front: /user/login
+  front: /front
```

Comment:
Can we use `/` as the front page or does Drupal force us to use a non-empty path?